README-dev.rst

```diff
@@ -13,7 +15,8 @@ Releasing
 * Tag the release.  For example: ``git tag -a 1.0.0 -m 'version 1.0.0'``
 * Push the tag and new ``master``: ``git push origin 1.0.0 ; git push origin master``
 * Update the `python-driver` submodule of `python-driver-wheels`,
-  commit then push. This will trigger TravisCI and the wheels building.
+  commit then push.
+* Trigger the Github Actions necessary to build wheels for the various platforms
```

Comment:
The wheel building process is now managed by Github Actions rather than Travis CI.  This should be addressed in a more comprehensive way when python-driver-wheels is donated.